lib/demo.rb

```diff
@@ -0,0 +1,63 @@
+###### This file is without connection Stack and Queue classes ############
+
+# require './Stack.rb'
+# require './Queue.rb'
+
+class JobSimulation
+  attr_reader :workers, :waiting, :roll, :fired, :hired
```

Comment:
You shouldn't add more variables here that are exposed to the 'outside' since I didn't tell you you could do that. It changes the functionality/ability of this class.